### PR TITLE
VSTS-270 Revert Scanner for .NET to 5.4.1.41282

### PR DESCRIPTION
**Pull request by mickael-caro-sonarsource**
Please review our [contribution guidelines](https://github.com/SonarSource/sonar-scanner-vsts/blob/master/contributing.md).

And please ensure your pull request adheres to the following guidelines: 

- [ ] Please explain your motives to contribute this change: what problem you are trying to fix, what improvement you are trying to make
- [ ] Use the following formatting style: [SonarSource/sonar-developer-toolset](https://github.com/SonarSource/sonar-developer-toolset#code-style)
- [ ] Provide a unit test for any code you changed
- [ ] If there is a [JIRA](https://jira.sonarsource.com/browse/VSTS) ticket available, please make your commits and pull request start with the ticket ID (VSTS-XXXX)

### CODE REVIEW

File: extensions/sonarqube/vss-extension.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "manifestVersion": 1,
   "id": "sonarqube",
   "name": "SonarQube",
-  "version": "5.3.0",
```
Comment (andrei-epure-sonarsource) [inline extensions/sonarqube/vss-extension.json:5]: Shouldn't this be a bugfix release?

**Review by csaba-sagi-sonarsource — APPROVED** [review]
LGTM!

File: extensions/sonarqube/vss-extension.json (hunk 1 above)
Comment (csaba-sagi-sonarsource) [reply extensions/sonarqube/vss-extension.json:5]: I think the bugfix release will be the one once we really fix the scanner, not?

Comment (csaba-sagi-sonarsource) [reply extensions/sonarqube/vss-extension.json:5]: BTW If we wait like an hour or to, the scanner bugfix release might be done and we could embed that version of the scanner right away?

Comment (mickael-caro-sonarsource) [reply extensions/sonarqube/vss-extension.json:5]: Yes you're right, i'm fixing it.

Comment (andrei-epure-sonarsource) [reply extensions/sonarqube/vss-extension.json:5]: @csaba-sagi-sonarsource I would rather mitigate the issue ASAP and we can do the bugfix release of the scanner also ASAP but this is affecting users as we speak so 1h matters

**Review by andrei-epure-sonarsource — APPROVED** [review]
Lgtm

File: extensions/sonarqube/vss-extension.json (hunk 1 above)
Comment (mickael-caro-sonarsource) [reply extensions/sonarqube/vss-extension.json:5]: We just have to think of the cache that Azure Pipelines apply on the extensions : if there are too many release in a short period of time, it takes time before it's actually rolled out everywhere.

Comment (andrei-epure-sonarsource) [reply extensions/sonarqube/vss-extension.json:5]: And many users cannot downgrade so are blocked

Comment (andrei-epure-sonarsource) [reply extensions/sonarqube/vss-extension.json:5]: There is no hurry for the AZDO 2nd release, and it should only be done in a morning (we need a postportem for this).

I meant that we need to do the bugfix release for the S4NET , for the people manually using the S4NET in their pipelines.

**Pull request merged**